lib/widgets/app.dart

```diff
@@ -94,6 +94,36 @@ class ZulipApp extends StatelessWidget {
 
         shape: Border(bottom: BorderSide(color: Color(0xffcccccc))),
       ),
+      elevatedButtonTheme: ElevatedButtonThemeData(
+        // Apply letter spacing to the text style for elevated buttons
+        style: ButtonStyle(
+          textStyle: MaterialStateProperty.all<TextStyle>(
+            const TextStyle(
+              letterSpacing: 0.01,
```

Comment:
> The letterSpacing  is set to 0.01 times the font size

If you look at the documentation for `TextStyle.letterSpacing`, what does it say about what the value means?

Reply:
Sorry to as @gnprice  Sir
Could you please provide a reference to this.
I think it would do me a lot good

Reply:
Flutter's API reference docs are at: https://api.flutter.dev/

The search box there should be able to take you to the `TextStyle` class and its `letterSpacing` property. That's the property that's being set by this line of code.

What does the documentation for that property say about what the property's value means?